app/uk/gov/hmrc/softdrinksindustrylevystub/services/SubscriptionGenerator.scala

```diff
@@ -22,15 +22,152 @@ import cats.implicits._
 import org.scalacheck._
 import org.scalacheck.support.cats._
 import uk.gov.hmrc.smartstub._
-import uk.gov.hmrc.softdrinksindustrylevystub.models._
+import uk.gov.hmrc.softdrinksindustrylevystub.models.internal._
+import uk.gov.hmrc.softdrinksindustrylevystub.models.maxL
+import ActivityType._
+import uk.gov.hmrc.softdrinksindustrylevystub.models.CreateSubscriptionResponse
+import uk.gov.hmrc.softdrinksindustrylevystub.models.EnumUtils.idEnum
 
 object SubscriptionGenerator {
 
+  lazy val store: PersistentGen[String, Option[Subscription]] = genSubscription.asMutable[String]
+
+  def genSubscription: Gen[Option[Subscription]] = {
```

Comment:
I recommend using a predetermined convention here rather than a generator for Option[Subscription].
For example making all UTR's starting with a certain character be blank by default in the store.
This might make it easier for Yordan to test with.

I don't think this should block merge and deploy though, could be a separate ticket.